src/System.Net.Http/tests/FunctionalTests/System.Net.Http.Tests.csproj

```diff
@@ -19,6 +19,9 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Windows_Debug|AnyCPU' " />
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Windows_Release|AnyCPU' " />  
   <ItemGroup>
+    <Compile Include="$(CommonTestPath)\System\Net\HttpTestServers.cs">
+      <Link>Common\System\Net\HttpTestServers.cs</Link>
```

Comment:
@josguil I think we should have a larger discussion regarding configuration file locations and how these can be changed by the users to allow them to exercise the code against their own local servers.